forge/context-driver/memory.js

```diff
@@ -1,15 +1,55 @@
+const { getObjectProperty, getItemSize } = require('./quotaTools')
 const util = require('@node-red/util').util
 
 const store = {}
-
+let app
 module.exports = {
-    init: function (app) {
+    init: function (_app) {
+        app = _app
         app.log.info('FlowForge File Server using Memory Context')
     },
     set: async function (projectId, scope, input) {
+        if (!store[projectId]) {
+            store[projectId] = {}
+        }
+        const projectStore = store[projectId]
+        const quotaLimit = app.config?.context?.quota || 0
```

Comment:
@hardillb Do we need a safe default here?

Reply:
Same as before "We need an option to have no limits"

Reply:
OK, so if we leave this as is, then that is catered for already (line below checks that `quotaLimit > 0` before doing any checks